### PR TITLE
organization edit page

### PR DESCRIPTION
**Pull request by AjeyataVerma**
https://trello.com/c/NouABWL2/247-super-admin-organization-edit-page
Screenshot from 2022-03-02 16-55-20

### CODE REVIEW

File: src/sagas/rootSagas.js [hunk 1]
```diff
@@ -93,6 +94,10 @@ export default function* rootSagas() {
     mcqQuestionSaga(),
     getquestionsSaga(),
     getMcqCountSaga(),
+<<<<<<< HEAD
+    editOrganizationWatcherSaga(),
+=======
```
Comment (somnath77) [inline src/sagas/rootSagas.js:99]: remove these buggy lines 97, 99, 101

**Comment by pooja-mane** [comment]
Please add screenshot for edit form

File: src/modules/superAdmin/organization/OrganizationComponent/OrganizationComponent.jsx [hunk 2]
```diff
@@ -92,7 +90,6 @@ OrganizationComponent.propTypes = {
   //   onClickEdit: PropTypes.func.isRequired,
```
Comment (Durgesh00) [inline src/modules/superAdmin/organization/OrganizationComponent/OrganizationComponent.jsx:90]: remove Commented code.

File: src/modules/superAdmin/organization/OrganizationComponent/OrganizationComponent.jsx [hunk 3]
```diff
@@ -35,9 +34,8 @@ const OrganizationComponent = ({
           >
             <ActionTile
               id={organization.id}
-              className='mb-3'
+              className='mb-3 ml-auto class  with mb-3'
```
Comment (somnath77) [inline src/modules/superAdmin/organization/OrganizationComponent/OrganizationComponent.jsx:37]: it should be just `className='mb-3 ml-auto'`

Comment (AjeyataVerma) [reply src/modules/superAdmin/organization/OrganizationComponent/OrganizationComponent.jsx]: yes corrected

File: src/modules/superAdmin/organizationEdit/OrganizationEditContainer/index.jsx [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+      dispatch(editOrganizationClearAction());
+      setEditOrganization({ type: 'clear' });
+    };
+  }, []);
+
+  const handleOrganizationNameChange = useCallback(
+    (event) => {
+      const name = event.target.value;
+      setEditOrganization({
+        type: 'nameErrTxt',
```
Comment (somnath77) [inline src/modules/superAdmin/organizationEdit/OrganizationEditContainer/index.jsx:86]: Try to clear `nameErrTxt` in `type: 'name'` reducer method. same for all handle methods

File: src/modules/superAdmin/organizationEdit/OrganizationEditContainer/index.jsx [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+      },
+    } = editOrganization;
+
+    setEditOrganization({
+      type: 'resetCountErrs',
+    });
+    const validData = {
+      name,
+      description,
+      organization_id: 1,
```
Comment (somnath77) [inline src/modules/superAdmin/organizationEdit/OrganizationEditContainer/index.jsx:154]: organization_id is unnecessary right ? is so, remove this

File: src/modules/superAdmin/organizationEdit/OrganizationEditContainer/index.jsx [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+      allow_mcq: allowMcq,
+    };
+    schema.isValid(validData).then(async (valid) => {
+      if (!valid) {
+        validateData(schema, validData, setEditOrganization);
+      } else {
+        dispatch(editOrganizationRequestAction({ putData }));
+      }
+    });
+    history.push(ROUTES.SUPER_ADMIN + SUPER_ADMIN_ROUTES.ORGANIZATIONS);
```
Comment (somnath77) [inline src/modules/superAdmin/organizationEdit/OrganizationEditContainer/index.jsx:175]: destructure `ROUTES` and  `SUPER_ADMIN_ROUTES` above and then use it like
```
const { SUPER_ADMIN } = ROUTES;
const { ORGANIZATIONS } = SUPER_ADMIN_ROUTES;
```

File: src/modules/superAdmin/organizationEdit/OrganizationEditContainer/index.jsx [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+  };
+
+  const finishOrganizationEdit = useCallback(() => {
+    history.push(ROUTES.SUPER_ADMIN + SUPER_ADMIN_ROUTES.ORGANIZATIONS);
```
Comment (somnath77) [inline src/modules/superAdmin/organizationEdit/OrganizationEditContainer/index.jsx:179]: same as above

**Review by somnath77 — APPROVED** [review]

**Review by pooja-mane — APPROVED** [review]

**Pull request merged**